src/components/ChannelListMessenger.js

```diff
@@ -98,6 +110,9 @@ const ChannelListMessenger = withChatContext(
 
     renderChannels = () => (
       <FlatList
+        ref={(flRef) => {
+          this.props.setFlatListRef && this.props.setFlatListRef(flRef);
+        }}
```

Comment:
not sure how you feel about optional chaining 
```suggestion
        ref={(flRef) => this.props?.setFlatListRef?.(flRef)}
```

Reply:
@vir I will take care of this in coming release. I will need to do some testing around this, don't want to make it a blocker for this release :)